device/inventec/x86_64-inventec_d6556-r0/installer.conf

```diff
@@ -0,0 +1,4 @@
+CONSOLE_PORT=0x3f8
+CONSOLE_DEV=0
+CONSOLE_SPEED=115200
```

Comment:
this is no longer needed.

https://github.com/Azure/sonic-buildimage/commit/fd5a3cf6fbcb16fdb20ae89045fc4eb2f5187ee6

Reply:
Agree to have set_if_type in APP_DB and incorporate with xrcvd daemon, we will study how to do it in the next version. Removing CONSOLE_PORT/CONSOLE_DEV/CONSOLE_SPEED will be included in the next veresion as well. Thanks for the information and the idea.